src/Containers.hs

```diff
@@ -159,9 +161,9 @@ instance Container IS.IntSet where
 -- Additional operations that don't make much sense for e.g. Maybe
 ----------------------------------------------------------------------------
 
--- | A class for 'Container's that aren't trivial like 'Maybe' (e.g. can hold
+-- | A class for 'ToList's that aren't trivial like 'Maybe' (e.g. can hold
 -- more than one value)
-class Container t => NontrivialContainer t where
+class ToList t => Container t where
```

Comment:
@Martoon-00 asked to save type aliases for previous names. So could you please add `type NontrivialContainer = Container` and export this? This can help with backwards compatibility for a while.